mlflow-tests/tests/validations/mcp_validations.py

```
@@ -0,0 +1,137 @@
+        test_context: Test context containing deleted server name.
+
+    Raises:
+        AssertionError: If server deletion verification fails.
+    """
+    logger.info(f"Validating MCP server deletion for user '{test_context.active_user.uname}' in workspace '{test_context.active_workspace}'")
+
+    # Validate no error occurred
+    if test_context.last_error is not None:
+        error_response: ErrorResponse = test_context.last_error
+        logger.error(f"Validation failed: MCP server deletion encountered an error for user '{test_context.active_user.uname}': {error_response.error.code} - {error_response.error.message}")
+        assert False, \
```

Ruff (0.16.1)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)